### PR TITLE
cp sourcing related icons

### PR DESCRIPTION
**Pull request by ChaitanyaGadodia**

### CODE REVIEW

**Comment by zhirzh** [comment]
some duplicates or alts:

- chevron-down: arrow-down, arrow-drop-down
- delete-2: delete, delete-filled
- edit-alt: edit-note
- face-to-face-3: team
- filter-2: control-unit
- note-alt: note, note-1
- site-visit-alt: business

File: packages/pebble-shared/svgs/added-details.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg width="1000" height="1000" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M83.333 250h500v83.333h-500V250zm0 166.667h500V500h-500v-83.333zm0 166.666h333.333v83.334H83.333v-83.334zm583.75 125l-125.417-125-62.5 62.5 187.917 187.5 291.25-291.666-62.5-62.5-228.75 229.166z" fill="#000"/></svg>
```
Comment (zhirzh) [inline packages/pebble-shared/svgs/added-details.svg:1]: most svgs are missing `width` and `height` attrs

**Review by zhirzh — CHANGES_REQUESTED** [review]

**Comment by zhirzh** [comment]
new icons are heavily based on material icons. it's better to use their standardized names rather than adding suffixes like `xxx-2` or `xxx-alt`

some examples:

- site-visit-alt: [business](https://material.io/resources/icons/?search=business&icon=business&style=baseline)
- disabled: [person-add-disabled](https://material.io/resources/icons/?search=person&icon=person_add_disabled&style=baseline)
- edit-alt: [border-color](https://material.io/resources/icons/?search=pencil&icon=border_color&style=baseline)

File: packages/pebble-shared/svgs/added-details.svg (hunk 1 above)
Comment (ChaitanyaGadodia) [reply packages/pebble-shared/svgs/added-details.svg:1]: added.

**Review by zhirzh — APPROVED** [review]

**Comment by l3satwik** [comment]
@ChaitanyaGadodia some icons are unnecessary, can be used by rotating existing icons (like, dots, down etc.)
Also run the storybook on local and check the icons. (cloud-upload.svg looks smaller than rest). Better to revert this PR and fix these.

**Comment by ChaitanyaGadodia** [comment]
@l3satwik the only existing icons which could have been used was `arrow-down` but it has a different width, style. Also cloud upload doesn't have any extra padding have checked for it.

**Pull request merged**